plugin/fzf.vim

```diff
@@ -455,7 +463,8 @@ function! s:pushd(dict)
       return 1
     endif
     let a:dict.prev_dir = cwd
-    execute 'lcd' s:escape(a:dict.dir)
+    let dir = s:escape(a:dict.dir)
+    execute 'lcd' has('win32unix') ? s:escape(s:cygpath(dir)) : dir
```

Comment:
`fzf#shellescape` isn't used in `s:cygwin` even though it uses `system`. To be consistent with recent changes, I'll use `fzf#shellescape` in `s:cygwin`.